### ISSUE TITLE
Compile error using project list to load 9c329b4 into v0.1.1-alpha (updating Rowan using Rowan)

### ISSUE THREAD

**Issue by dalehenrich** [dalehenrich/Rowan#198]
The troublesome class is **RwPackageProjectImageSpecification** and it is not in the `tempSymbols` for **RwGsPatchSet** and is the source of the compile error below. After a little analysis, it turns out the class *is* in the `classesWithNewVersions` for **RwGsPatchSet** (one of it's super classes had an instance variable added) ... we're obviously missing a test case for this ...

My plan is to branch the bugfix for this issue at 53ec253 (the commit before 9c329b4), so that I can use my `rowan_update_3215` stone (where I found this bug) to validate the proposed fix ... then once confirmed, I can go back to 53ec253 (at my leisure:) and create a test case that duplicates the bug.

Here's the stack:
```
a CompileError occurred (error 1001), compilation errors -- parameter 1 is error descriptor
---
CompileError (AbstractException) >> _signalWith: @5 line 25
CompileError (AbstractException) >> signal @2 line 47
RwPackageReadWriteTest class (Behavior) >> _checkCompileResult:source:suppressCompileWarning: @11 line 21
RwPackageReadWriteTest class (Behavior) >> _checkCompileResult:source: @2 line 3
RwPackageReadWriteTest class (Behavior) >> compileMethod:dictionaries:category:intoMethodDict:intoCategories:intoPragmas:environmentId: @3 line 34
RwGsMethodSourceSymbolDictPatch (RwGsMethodPatch) >> compileUsingNewClasses:andExistingClasses: @17 line 12
RwGsPatchSet >> compileMethodPatch: @2 line 3
[] in  ExecBlock1 (RwGsPatchSet) >> compileMethods @2 line 5
[] in  ExecBlock1 (Set) >> do: @2 line 11
[] in  ExecBlock2 (KeyValueDictionary) >> keysDo: @2 line 7
KeyValueDictionary >> keysAndValuesDo: @18 line 15
KeyValueDictionary >> keysDo: @2 line 6
Set >> do: @3 line 11
RwGsPatchSet >> compileMethods @4 line 5
RwGsPatchSet >> apply @7 line 11
RwGsPatchSet >> applyForNewClassVersions: @8 line 17
RwGsImage class >> applyModification:visitorClass:instanceMigrator: @20 line 24
RwGsImage class >> applyModification:instanceMigrator: @7 line 8
RwPrjLoadTool >> loadProjectSetDefinition:instanceMigrator: @13 line 13
RwPrjLoadTool >> loadProjectSetDefinition: @4 line 4
[] in  ExecBlock0 (RwPrjLoadTool) >> loadProjectNamed:withConfiguration: @2 line 9
ExecBlock0 (ExecBlock) >> on:do: @3 line 42
RwPrjLoadTool >> loadProjectNamed:withConfiguration: @6 line 10
[] in  ExecBlock0 (RowanProjectService) >> loadProjectNamed: @4 line 3
ExecBlock0 (ExecBlock) >> on:do: @3 line 42
RowanProjectService >> loadProjectNamed: @3 line 4
GsNMethod class >> _gsReturnToC @1 line 1
```

**Comment by dalehenrich** [comment]
Here's the stack for `testIssue198A`. Note that the call to `RwGsPatchSet >> updateSymbolAssociations` is made from line 23 of `RwGsPatchSet >> apply`, while in the original stack the method `RwGsPatchSet >> compileMethods` is called from line 11 of `RwGsPatchSet >> apply`, so `testIssue198A` is presumably an alternate failure mechanism for the same bug:
```
UserDefinedError (AbstractException) >> _signalWith: @5 line 25
UserDefinedError (AbstractException) >> signal @2 line 47
RwGsSymbolDictionaryRegistry (Object) >> error: @6 line 7
[] in  ExecBlock0 (RwGsSymbolDictionaryRegistry_Implementation class) >> _loadedClassFor:noNewVersion:newVersion:instance: @5 line 10
IdentityKeyValueDictionary >> at:ifAbsent: @25 line 19
RwGsSymbolDictionaryRegistry_Implementation class >> _loadedClassFor:noNewVersion:newVersion:instance: @7 line 7
RwGsSymbolDictionaryRegistry_Implementation class >> addNewClassVersionToAssociation:instance: @2 line 6
RwGsSymbolDictionaryRegistry >> addNewClassVersionToAssociation:implementationClass: @2 line 7
RwGsClassVersioningSymbolDictPatch >> installNewClassVerionInSystem @3 line 6
[] in  ExecBlock1 (RwGsPatchSet) >> updateSymbolAssociations @2 line 4
[] in  ExecBlock1 (Set) >> do: @2 line 11
[] in  ExecBlock2 (KeyValueDictionary) >> keysDo: @2 line 7
KeyValueDictionary >> keysAndValuesDo: @23 line 20
KeyValueDictionary >> keysDo: @2 line 6
Set >> do: @3 line 11
RwGsPatchSet >> updateSymbolAssociations @2 line 4
RwGsPatchSet >> apply @17 line 23
RwGsPatchSet >> applyForNewClassVersions: @8 line 17
RwGsTestImage class (RwGsImage class) >> applyModification:visitorClass:instanceMigrator: @20 line 24
RwGsTestImage class (RwGsImage class) >> applyModification:instanceMigrator: @8 line 13
RwPrjLoadTool >> loadProjectSetDefinition:instanceMigrator: @13 line 13
RwPrjLoadTool >> loadProjectSetDefinition: @4 line 4
RwPrjLoadTool >> loadProjectDefinition: @5 line 7
RwRowanProjectIssuesTest >> testIssue198A @37 line 57
RwRowanProjectIssuesTest (TestCase) >> performTest @3 line 2
[] in  ExecBlock0 (TestCase) >> runCase @3 line 5
ExecBlock0 (ExecBlock) >> ensure: @2 line 12
RwRowanProjectIssuesTest (TestCase) >> runCase @4 line 6
[] in  ExecBlock0 (TestCase) >> debug @4 line 2
ExecBlock0 (ExecBlock) >> ensure: @2 line 12
RwRowanProjectIssuesTest (TestCase) >> debug @2 line 3
RwRowanProjectIssuesTest class (TestCase class) >> debug: @3 line 3
[] in  ExecBlock1 (JadeServer) >> sbRunMethodTests: @3 line 6
Array (Collection) >> do: @5 line 10
JadeServer64bit32 (JadeServer) >> sbRunMethodTests: @8 line 6
GsNMethod class >> _gsReturnToC @1 line 1
```

**Comment by dalehenrich** [comment]
confirmed fix by loading a5ee3b9, add iv to **RwProjectSpecification** then loading a87f369 (which has package move)
